test/Gradient/Gradients.C

```diff
@@ -6,11 +6,11 @@
 #include "clad/Differentiator/Differentiator.h"
 #include <cmath>
 
-double f_add1(double x, double y) {
+__attribute__((always_inline)) double f_add1(double x, double y) {
```

Comment:
To test the `hasAttribute` function: 
```suggestion
__attribute__((always_inline)) double f_add1(double x, double y);
__attribute__((always_inline)) double f_add1(double x, double y) {
```